src/api/data/commands.yaml

```diff
@@ -2492,6 +2493,16 @@ selections.select.orLeap:
     qwerty: |-
       `s` (kakoune: normal)
 
+selections.select.orSneak:
+  title:
+    en: Sneak or select
+
+  commands: |-
+    [".ifEmpty", { then: [[".seek", { inputLength: 2, $exclude: [] }]], otherwise: [[".selections.select", { $exclude: [] }]] }]
+
+  keys:
+    qwerty: ""
```

Comment:
I'm guessing this is a result of having an initial command and then removing it (the script that generates this file tries to keep old values to avoid throwing away manual changes). This should be removed (and ditto a bit lower in the file).